web-api/src/business/useCaseHelper/generateChangeOfAddressHelper.ts

```diff
@@ -126,12 +126,19 @@ export const generateChangeOfAddressHelper = async ({
       `"change-of-address-job|${jobId}" job finished`,
     );
 
+    let resultsUser: RawPractitioner = user;
     if (websocketMessagePrefix === 'user') {
+      const liveUser = await applicationContext
+        .getPersistenceGateway()
+        .getUserById({ applicationContext, userId: user.userId });
+
       const userEntity = new Practitioner({
-        ...user,
+        ...liveUser,
+        contactInfo,
         isUpdatingInformation: false,
       });
 
+      resultsUser = userEntity.validate().toRawObject();
```

Comment:
Is there any reason not to just get the most recent user data regardless of the `websocketMessagePrefix` condition?

Reply:
seems like 'admin' call does not use the user information being sent back so I think it is safe to add it there too. Thanks for the feedback!